index.js

```diff
@@ -45,52 +50,85 @@ function getLatestData(caseData) {
 }
 
 async function getCovidData(channelId, query) {
-  query = isEmpty(query) ? 'all' : query;
-  if (query == 'all') {
+  console.log(query);
+  query = isEmpty(query) ? ['all'] : (Array.isArray(query) ? query : [query]);
+  if (query[0] == 'all') {
     const { data } = await axios.get('https://corona.lmao.ninja/all');
     try {
-      var cases = data.cases;
-      var deaths = data.deaths;
-      var recovered = data.recovered;
+      var cases = (data.cases).toLocaleString('en');
+      var deaths = (data.deaths).toLocaleString('en');
+      var recovered = (data.recovered).toLocaleString('en');
       var dateLastUpdated = moment(data.updated).format('MMMM D, YYYY [at] h:mm a');
```

Comment:
We don't need `toLocaleString` on these since they're just numbers. And `moment` will handle the date conversion just fine. JS will handle the string conversion on `console.log` automatically, so you can just destructure out the variables like this:

```suggestion
      const { cases, deaths, recovered, updated } = data;
      const dateLastUpdated = moment(updated).format('MMMM D, YYYY [at] h:mm a');
```

`const` by default is also the new norm, unless you need mutability, in which you would use `let`. It's fairly recent ES6 stuff though, and not the hugest deal.